src/main/java/com/emonadeo/autorun/AutoRunModMenu.java

```diff
@@ -18,22 +18,22 @@ public ConfigScreenFactory<Screen> getModConfigScreenFactory() {
     public Screen create(Screen screen) {
         ConfigBuilder builder = ConfigBuilder.create()
                 .setParentScreen(screen)
-                .setTitle(new TranslatableText("title." + AutoRun.MODID + ".config"));
+                .setTitle(Text.translatable(("title." + AutoRun.MODID + ".config")));
```

Comment:
This is required due to the text changed described in the new version of Fabric [here](https://fabricmc.net/2022/06/07/119.html)